webrtc/RTCPeerConnection-setRemoteDescription.html

```diff
@@ -47,6 +47,43 @@
       assert_unreached('Error ' + e.name + ': ' + e.message);
     }));
   }, 'Triggers ontrack when called with a remote description and the MSID is parsed.');
+
+  promise_test(() => {
+    const pc = new RTCPeerConnection();
+    return pc.setRemoteDescription(
+      {'type': 'offer', 'sdp': 'bogus'})
+    .then(() => {
+      assert_unreached('Bogus SDP should not be accepted');
+    })
+    .catch(e => {
+      assert_equals(e.name, 'InvalidAccessError');
```

Comment:
https://w3c.github.io/webrtc-pc/#set-description says "If the content of *description* is invalid", but doesn't define the type of the *description* argument or what "the content" is. Does it mean simply the sdp member of a RTCSessionDescriptionInit  dictionary? Can you file spec issues to clarify?